test/integration/visitor_can_view_items_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class VisitorView < ActionDispatch::IntegrationTest
+
+  test 'site visitor can view stickers' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.